gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -481,11 +480,15 @@ def run(argv=None):
         schema_converter.convert_table_schema_to_json_bq_schema(schema))
     with filesystems.FileSystems.create(schema_file) as file_to_write:
       file_to_write.write(schema_json)
+    known_args.schema_json = schema_file
```

Comment:
I'd rather have this var as `schema_json_file` or `schema_file` - it's not immediately clear that the value is not holding actual schema, especially since before a similar var was doing it. Also maybe we shouldn't overpopulate known_args - schema is used only once AFAICS, so lets keep it as single var.

Reply:
As you advised I am sending schema_file directly to the method, so no need to add to known_args anymore.